consensus/src/validation/step.rs

```diff
@@ -121,29 +121,29 @@ impl<T: Operations + 'static, D: Database> ValidationStep<T, D> {
 
     async fn verify_candidate(
         candidate: &Block,
-        state_root: StateRoot,
+        prev_state: StateRoot,
         executor: Arc<T>,
         expected_generator: PublicKeyBytes,
     ) -> Result<(), OperationError> {
         let header = candidate.header();
 
         // Verify faults
         executor
-            .verify_faults(header.height, candidate.faults())
+            .validate_faults(header.height, candidate.faults())
             .await?;
 
         // Verify candidate header
-        let (_, voters, _) = executor
-            .verify_candidate_header(header, &expected_generator)
+        let (_, cert_voters, _) = executor
+            .validate_block_header(header, &expected_generator)
             .await?;
 
         // Verify state transition
-        let vst_result = executor
-            .verify_state_transition(state_root, candidate, &voters)
+        let transition_result = executor
+            .validate_state_transition(prev_state, candidate, &cert_voters)
             .await?;
```

Comment:
Nit: function name was renamed, yet comment stayed the same, is this by design so? Similar in other places.